appservice/src/remoteDebug/startRemoteDebug.ts

```diff
@@ -124,3 +110,38 @@ async function getDebugConfiguration(context: IActionContext, language: RemoteDe
 
     return config;
 }
+
+function getPythonDebugConfiguration(sessionId: string, portNumber: number, host: string): vscode.DebugConfiguration {
+    const config: vscode.DebugConfiguration = {
+        name: sessionId,
+        type: 'python',
+        request: 'attach',
+        connect: {
+            host: host,
+            port: portNumber,
+        },
+        pathMappings: [
+            {
+                localRoot: '${workspaceFolder}',

```

Comment:
This doesn't work for multi-root workspaces. The logic around `workspaceFolders` which you put into `getNodeDebugConfiguration` should still be used for both node _and_ python
![Screen Shot 2021-08-10 at 11 02 08 AM](https://user-images.githubusercontent.com/11282622/128912137-975b19a8-7f83-4ada-a173-1ac083884c90.png)

Reply:
I've refactored this to allow multi root workspaces and show a pick list when there is more than one instead of just throwing an error